### PR TITLE
20759566: Added Featured Job Section Component

### PR DESCRIPTION
**Pull request by timmy38233**
This PR is dependent on finishing the global styles first

### CODE REVIEW

File: src/components/container/FeaturedJobs/FeaturedJobs.js
Comment (jasper-prgh) [inline src/components/container/FeaturedJobs/FeaturedJobs.js]: Please use camelCase over PascalCase for asset file names:

e.g. `viewMore.svg` rather than `ViewMore.svg`

**Review by jasper-prgh — CHANGES_REQUESTED** [review]

File: src/components/container/FeaturedJobs/FeaturedJobs.js
Comment (jasper-prgh) [inline src/components/container/FeaturedJobs/FeaturedJobs.js:16]: I love this! But one last improvement:

In BEM-classes only the first element should be in PascalCase, so for example

instead of `FeaturedJobs__ViewMoreLink__Text` it should be `FeaturedJobs__viewMoreLink__text`

**Review by jasper-prgh — CHANGES_REQUESTED** [review]

File: src/components/container/FeaturedJobs/FeaturedJobs.js
Comment (timmy38233) [reply src/components/container/FeaturedJobs/FeaturedJobs.js:16]: Thanks for the hint, I did not know that :)

File: src/components/container/FeaturedJobs/FeaturedJobs.scss
Comment (jasper-prgh) [inline src/components/container/FeaturedJobs/FeaturedJobs.scss]: Won't approve this PR yet. Waiting for the global styles to be finished and adopted :) Great job though!

**Review by anna-bc — COMMENTED** [review]
Looks great. But I'm not sure if we should already approve it if the global styles are not yet finished and added?
@jasper-prgh

File: src/components/container/FeaturedJobs/FeaturedJobs.scss
Comment (tinkabel85) [inline src/components/container/FeaturedJobs/FeaturedJobs.scss]: this is a global variable, you can use it now:
--color-text: #7d7cac;

as well as for the h2 - the font color should be var(--color-bunting);

File: src/components/container/FeaturedJobs/FeaturedJobs.scss
Comment (anna-bc) [inline src/components/container/FeaturedJobs/FeaturedJobs.scss]: I don't think we have such a global variable

Comment (timmy38233) [reply src/components/container/FeaturedJobs/FeaturedJobs.scss]: You're right! I thought we might use one in the future, but I removed it for now

**Review by anna-bc — APPROVED** [review]
Looks good!

**Review by jasper-prgh — APPROVED** [review]

**Comment by timmy38233** [comment]
Still dependent on approval of PR #21. After that is merged, I will update this accordingly

**Review by tinkabel85 — APPROVED** [review]

**Review by jasper-prgh — APPROVED** [review]

**Pull request merged**
